### DISCUSSION TITLE
Separate boundary ids for different blocks in the mesh

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#23051] [Q&A Modules: General] [answered]
Hello MOOSE Experts,

There are two materials in my mesh and I want to separate the boundary ids for both the domains.
Can you please suggest, how should I proceed?

Best,
Avtar

**Comment by GiudGiud** [comment, marked as answer]
Hello

I think you are looking for this mesh generator
https://mooseframework.inl.gov/source/meshgenerators/BreakBoundaryOnSubdomainGenerator.html

The materials are defined on different subdomains right?

Guillaume

**Reply by avtarsinghh1991** [reply]
Yes, materials are defined on different subdomains.
Thank you very much.

Best,
Avtar
